### PR TITLE
Adding content for "this" special attribute.

### PR DESCRIPTION
**Pull request by Serdaro**
https://github.com/hazelcast/hazelcast-reference-manual/issues/562

### CODE REVIEW

File: src/docs/asciidoc/distributed_query.adoc [hunk 1]
```diff
@@ -388,6 +388,25 @@ NOTE: Usage of this system property is supported for BINARY and OBJECT in-memory
...
+```
+
+Another basic sample using `SqlPredicate` is shown below.
+
+```
+new SqlPredicate("this = 'jones'")
+new SqlPredicate("this.age > 33")
+```
+
+The special attribute `this` acts on the value of a map entry. Typically, you do not need to specify it while accessing a property of an entry's value, since its presence is implicitly assumed if the special attribute <<querying-entry-keys-with-predicates, __key>> is not specified. See the
```
Comment (kwart) [inline src/docs/asciidoc/distributed_query.adoc]: Unfinished sentence: `See the`

**Comment by Serdaro** [comment]
@kwart , At first I was structuring the sentence so that the reader will be directed to the "__key" content. I forgot that phrase after I made that section direction as an in-sentence cross-reference.

**Comment by kwart** [comment]
run-lab-run

**Comment by kwart** [comment]
run-lab-run

**Pull request merged**